docs/index.rst

```diff
@@ -1,31 +1,56 @@
-.. TabPy documentation master file, created by
-   sphinx-quickstart on Tue Mar 12 01:49:50 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
-Welcome to TabPy's documentation!
-=================================
-
-.. toctree::
-   :maxdepth: 2
-   :caption: Contents:
-
-   about.md
-   server-config.md
-   server-configurations.md
-   server-download.md
-   server-rest.md
-   server-startup.md
-   tabpy-tools.md
-   tabpy-virtualenv.md
-   TableauConfiguration.md
-   security.md
-
-
-
-Indices and tables
-==================
-
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
+TabPy
+=====
+
+TabPy (Tableau Python Server) is external server implementation which
+allows expanding Tableau with executing Python scripts on table
+calculation.
+
+
+ .. toctree::
+   :maxdepth: 2
+   :caption: General documentation:
+
+-  `About TabPy <about.md>`__
+-  `TabPy Server Download Instructions <server-download.md>`__

```

Comment:
Should only need one trailing underscore for these to render, though I think as long as the title of the target file matches the description you are looking for it will render that would automatically (and be preferable)